src/main/resources/messages_ru.properties

```diff
@@ -0,0 +1,57 @@
+invalid.id=Идентификатор не может быть отрицательным
+invalid.not-empty=${validatedValue} значение не может быть пустым
+invalid.not-null=${validatedValue} значение не может быть null
+invalid.size=${validatedValue} значение должно содержать от {min} до {max} символов
+invalid.max-size=${validatedValue} значение не может превышать {max} символов
+invalid.email=Пожалуйста, введите корректный адрес электронной почты (Пример: username@example.com)
+invalid.username=Пожалуйста, введите корректное имя пользователя, не используйте пробелы
+invalid.past-or-present.date=${validatedValue} дата не может быть будущей
```

Comment:
Sounds a bit weird to me. Maybe better be "дата не может быть в будущем"?

Reply:
We say "прошедшая дата, будущая дата" using "прошедшая", "будущая" as adjective. 
Maybe just leave only "неправильная дата", because all of these phrases are weird.